frontend/src/design/components/layout/DefaultSidebar.js

```diff
@@ -34,19 +34,22 @@ export const DefaultSidebar = ({ openDrawer, onOpenDrawerChange }) => {
     const datasetsSection = {
       title: 'Datasets',
       path: '/console/datasets',
-      icon: <FiPackage size={15} />
+      icon: <FiPackage size={15} />,
+      active: isModuleEnabled(ModuleNames.DATASETS)
     };
 
     const sharesSection = {
       title: 'Shares',
       path: '/console/shares',
-      icon: <ShareOutlined size={15} />
+      icon: <ShareOutlined size={15} />,
+      active: isModuleEnabled(ModuleNames.SHARES)
```

Comment:
Let me understand this a bit taking Shares as example. We use `isModuleEnabled` with the `Shares` module. But then, we do not check Shares active param in the config file, we directly check the Datasets active parameter. For me is a little misleading, I would expect that we verify the param of shares in config.json

Reply:
I believe there's no entry for `shares` in the `configs.json` file, and as we agreed before, `shares` should be enabled if and only if `datasets` module is enabled. I understand it's a bit confusing, but this is by far the best approach we can take for module enablement dependencies management in the frontend, because if we define every entry in the `configs.json` file and keep it up to the user to define which module should be enabled, we might end up with conflicting active statuses. wdyt?

Reply:
`shares` and `catalog` can be defined in the `config.json`, glossaries is part of catalog in the backend